src/app/pages/dashboard/widgets/network/widget-network/widget-network.component.ts

```diff
@@ -0,0 +1,212 @@
+import {
+  Component, ChangeDetectionStrategy, input,
+  computed,
+} from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import { UntilDestroy } from '@ngneat/until-destroy';
+import { TranslateService } from '@ngx-translate/core';
+import { ChartData, ChartOptions } from 'chart.js';
+import {
+  filter, map, shareReplay, skipWhile, switchMap,
+} from 'rxjs';
+import { KiB } from 'app/constants/bytes.constant';
+import { LinkState, NetworkInterfaceAliasType } from 'app/enums/network-interface.enum';
+import { buildNormalizedFileSize } from 'app/helpers/file-size.utils';
+import { BaseNetworkInterface, NetworkInterfaceAlias } from 'app/interfaces/network-interface.interface';
+import { WidgetResourcesService } from 'app/pages/dashboard/services/widget-resources.service';
+import { WidgetComponent } from 'app/pages/dashboard/types/widget-component.interface';
+import { SlotSize } from 'app/pages/dashboard/types/widget.interface';
+import { networkWidgetAspectRatio } from 'app/pages/dashboard/widgets/network/widget-network/widget-network.const';
+import { processNetworkInterfaces } from 'app/pages/dashboard/widgets/network/widget-network/widget-network.utils';
+import { LocaleService } from 'app/services/locale.service';
+import { ThemeService } from 'app/services/theme/theme.service';
+
+@UntilDestroy()
+@Component({
+  selector: 'ix-widget-network',
+  templateUrl: './widget-network.component.html',
+  styleUrls: ['./widget-network.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class WidgetNetworkComponent implements WidgetComponent {
+  size = input.required<SlotSize>();
+
+  protected isLoading = computed(() => !this.interface() || !this.interfaceUsage() || !this.reportingData());
+
+  protected interface$ = this.resources.networkInterfaces$.pipe(
+    skipWhile((state) => state.isLoading),
+    map((state) => processNetworkInterfaces(state.value)),
+    filter((interfaces) => interfaces.length > 0),
+    map(([firstInterface]) => firstInterface),
+    shareReplay({ bufferSize: 1, refCount: true }),
+  );
+
+  protected interface = toSignal(this.interface$);
+
+  protected interfaceUsage = toSignal(this.resources.realtimeUpdates$.pipe(
+    skipWhile(() => Boolean(!this.interface()?.name)),
+    map((update) => update.fields.interfaces),
+    map((interfaces) => interfaces?.[this.interface().name]),
+  ));
+
+  protected reportingData = toSignal(this.interface$.pipe(
+    filter(Boolean),
+    switchMap((networkInterface) => this.resources.networkInterfaceUpdate(networkInterface.name)),
+    map((response) => {
+      const updatedResponse = response[0];
+      (updatedResponse.data as number[][]).forEach((row, index) => {
+        // remove first column and convert kilobits/s to bytes
+        (updatedResponse.data as number[][])[index] = row.slice(1).map((value) => value * KiB);
```

Comment:
Network traffic must be calculated in bps (bits per second) and must use base-10 units (Kb, Mb, Gb) instead of base-2 units (KiB, MiB, GiB).